server/protocols/protocols-smtp/src/test/java/org/apache/james/smtpserver/SMTPServerTest.java

```diff
@@ -1285,7 +1287,7 @@ public void testAuth() throws Exception {
             .isFalse();
 
         smtpProtocol.sendCommand("AUTH PLAIN");
-        smtpProtocol.sendCommand(Base64.encodeAsString("\0" + noexistUserName + "\0pwd\0"));
+        smtpProtocol.sendCommand(Base64.getEncoder().encodeToString(("\0" + noexistUserName + "\0pwd\0").getBytes(UTF_8)));
```

Comment:
I'm not comfortable with this refactoring. It looks like there were specific steps in the Base64 impl you dropped like calling trim() for example.
BTW, I would like the deleted code to be well tested before being replaced to minimize the risk.

Reply:
Base64 is:
 - defined in tests
 - only used in tests, which did not broke
 - And we don't rely on that `trim()` feature.

I see no reasons to be keeping that freaks around.

However, I do agree that extra care is needed upon code removal.